main.go

```diff
@@ -47,9 +47,15 @@ var (
 	AFS       *afero.Afero = &afero.Afero{Fs: FS}
 )
 
+const (
+	AS_GIT_EDITOR = "as-git-editor"
+	ERROR_STRING  = "Error: %s"
+	SHIFT_TAB     = "shift+tab"
```

Comment:
constants in Go are usually not in uppercase, and barely never in "screaming snake case"

Reply:
I wasn't aware of this standard, thank you!